### ISSUE TITLE
Allow user to run html through a processor (ie. minifier)

### ISSUE THREAD

**Issue by cgmartin** [javoire/browserify-ng-html2js#17]
Feature request to allow a configurable means of processing the html before passing to ng-html2js

Usage idea/example:
```js
var gulp  = require('gulp'),
    browserify = require('browserify'),
    ngHtml2Js = require('browserify-ng-html2js'),
    source = require('vinyl-source-stream'),
    minify = require('html-minifier').minify; // https://www.npmjs.com/package/html-minifier

gulp.task('scripts', function() {
  return browserify('./src/app.js')
    .transform(ngHtml2Js({
      // ... other options ...
      htmlProcessor: // optional callback to process the html file before passing to ng-html2js
        function (htmlString) { 
          return minify(htmlString, {removeAttributeQuotes: true});
        }
    }))
    .bundle()
    .pipe(source('bundle.js'))
    .pipe(gulp.dest('./dist'));
});
```

I'm unsure if this could be supported when using browserify from the command-line or configured via package.json. Are callback functions able to be passed in as arguments somehow, or do alternate patterns exist? 

Or maybe there is a way to chain browserify transforms so that the html file can be processed prior to entering browserify-ng-html2js, or other workarounds I'm not considering?

Thanks!

**Comment by jamestalmage** [comment]
> Or maybe there is a way to chain browserify transforms so that the html file can be processed prior to entering browserify-ng-html2js, or other workarounds I'm not considering?

PR #25 will allow just that.

**Comment by cgmartin** [comment]
Sweet! Thanks, @jamestalmage

**Comment by jamestalmage** [comment]
@cgmartin looks like the fix published in `1.1.4`.
